### PR TITLE
Elfinder setup - pick up from the cleanup done by @Clindbergh

### PR DESCRIPTION
**Pull request by dbu**
continue the PR started in #616

### CODE REVIEW

**Comment by dbu** [comment]
fixed the versionadded statement.

@WouterJ does this look ok?

File: bundles/media/adapters/elfinder.rst [hunk 1]
```diff
@@ -16,62 +16,77 @@ Installation
...
+            <!-- app/config/config.xml -->
+            <?xml version="1.0" charset="UTF-8" ?>
+            <container xmlns="http://symfony.com/schema/dic/services">
+
+                <config xmlns="http://example.org/dic/schema/fm_elfinder">
+                    <instances
+                        locale="%locale%"
+                        editor="ckeditor"
+                    >
+                        <default>
```
Comment (WouterJ) [inline bundles/media/adapters/elfinder.rst]: The bundle does not yet support XML, that's why it fails. And btw, I don't think the configuration currently works as the authors think it should. I'll submit a PR this evening to add xML support. (I'm sorry to have completely ignored the docs the past months :flushed:)

File: bundles/media/adapters/elfinder.rst [hunk 2]
```diff
@@ -16,62 +16,77 @@ Installation
...
+                           ),
+                        ),
+                    ),
+                ),
+           ));
 
-           </container>
-
-       .. code-block:: php
+.. versionadded:: 2.0
```
Comment (WouterJ) [inline bundles/media/adapters/elfinder.rst]: versionaddeds do not allow empty lines (at least not on symfony.com)

File: bundles/media/adapters/elfinder.rst (hunk 1 above)
Comment (dbu) [inline bundles/media/adapters/elfinder.rst]: oh, ok. so i wait and update this again next week when we know what
exactly it looks like.

**Comment by dbu** [comment]
ok, just removed the empty line after versionadded

**Comment by dbu** [comment]
@WouterJ can we merge this? and should we put it in 1.2 or in master?

**Comment by dbu** [comment]
rebased after #673 fixed the yml example. @WouterJ can we merge this?

**Comment by dbu** [comment]
ping @WouterJ

**Pull request merged**